Mathlib/AlgebraicGeometry/Sites/Etale.lean

```diff
@@ -0,0 +1,38 @@
+/-
+Copyright (c) 2024 Christian Merten. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Christian Merten
+-/
+import Mathlib.AlgebraicGeometry.Morphisms.Etale
+import Mathlib.AlgebraicGeometry.PullbackCarrier
+import Mathlib.AlgebraicGeometry.Sites.BigZariski
+
+/-!
+
+# The étale site
+
+In this file we define the étale topology as a Grothendieck topology on the category
+of schemes.
```

Comment:
I think it would be good to explicitly mention the word "big" somewhere in this module docstring. And also in the docstrings below.